packages/react/react/src/components/Separated/Separated.tsx

```diff
@@ -15,7 +15,7 @@ export function Separated({ children, with: separator }: Props) {
       {childrenArray.map((child, i) => (
         <Fragment key={i}>
           {child}
-          {i + 1 !== childrenLength ? separator : null}
```

Comment:
ensures the separator is rendered between all children except after the last one. Essentially, for every child except the last, the separator will be displayed.

Contrastingly, the code proposed in your PR:
**would render the separator only after the last child**, altering the current visual representation of our UI. This change could have a varied impact on all instances where this code pattern is used, potentially disrupting the established and intended UI pattern across our application.

I also had a hard time understanding this part.
I think it would be better to write a test code and clearly understand the intention of the existing code.

Reply:
Thank you for your feedback. I didn't fully grasp the issue, and I will practice further review on the aspect I had in mind. This part has been restored in the following commit. Thank you.
[91bc089](https://github.com/toss/slash/pull/344/commits/91bc08979ef3130d8b10dcbc9ae7af97859f4a34)